### ISSUE TITLE
Control-C on command instructions loses cursor.

### ISSUE THREAD

**Issue by bschroeter** [google/python-fire#205]
When running a script with no arguments the user is presented with a man-style page listing the available commands. The natural way to dismiss this message is to either hit "q" or "control-c". The former works fine, but the latter triggers a KeyboardInterrupt which, when returning to the shell has lost the cursor and typed text no longer appears.

I've tested this on both MacOS and Linux, the same error occurs in both terminal and iterm2.

Take the following example:

```python
import fire

def myfunc():
    return True

if __name__ == '__main__':
    fire.Fire()
```

Run it with:

```shell
python test.py
```

Then hit control-c and try to type something in the returned shell. You'll see what I mean.

**Comment by dbieber** [comment]
> when returning to the shell has lost the cursor and typed text no longer appears.

yikes, sorry about that.

If you type reset it should fix things.
Marking this as a bug.

**Comment by bschroeter** [comment]
@dbieber, thanks for the quick response. Looking forward to the fix.
Cheers, B

**Comment by priyeshgpatel** [comment]
i like the version v0.1.3 type help, where it lists out all the options rather than man-style help screen.  wonder if you can add a feature to choose between the 2 styles.

**Comment by dbieber** [comment]
Fixed by 074f4b25906b4c7bc309a64feeba8873ec5e849c
